### PR TITLE
fix: add kingwisdom profile

### PR DESCRIPTION
**Pull request by kingwisdom**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/745"></a>

### CODE REVIEW

**Review by krishguptadev — CHANGES_REQUESTED** [review]
Can you reset the package-lock files and format the files to

```json
{
  "name": "Adeoye Temitayo",
  "bio": "Software Developer || Open Source Enthusiast.",
  "avatar": "https://avatars.githubusercontent.com/u/32174632?v=4",
  "links": [
    {
      "name": "Follow me on GitHub",
      "url": "https://github.com/kingwisdom",
      "icon": "github"
    },
    {
      "name": "Follow me on Twitter",
      "url": "https://twitter.com/adeoyeTemitayo9",
      "icon": "twitter"
    }
  ]
}

```

File: public/data/kingwisdom.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Adeoye Temitayo",
+    "bio": "Software Developer || Open Source Enthusiast.",
+    "avatar": "https://avatars.githubusercontent.com/u/32174632?v=4",
```
Comment (Panquesito7) [inline public/data/kingwisdom.json:4]:
```suggestion
    "avatar": "https://github.com/kingwisdom.png",
```

File: public/data/kingwisdom.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/adeoyeTemitayo9",
+        "icon": "twitter"
+      }
+    ]
+  }
+
```
Comment (Panquesito7) [inline public/data/kingwisdom.json:18]:
```suggestion
}
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by emmalearnscode — APPROVED** [review]

File: public/data/kingwisdom.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
```
Comment (Panquesito7) [inline public/data/kingwisdom.json:18]:
```suggestion
}
```

File: public/data/kingwisdom.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+        "url": "https://github.com/kingwisdom",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/adeoyeTemitayo9",
+        "icon": "twitter"
+      }
+    ]
+}
```
Comment (Panquesito7) [inline public/data/kingwisdom.json:17]:
```suggestion
}

```

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**